packages/fxa-auth-server/lib/senders/emails/templates/postChangePrimary/index.txt

```diff
@@ -0,0 +1,9 @@
+postChangePrimary-subject = "New primary email"
```

Comment:
This looks right, and maybe it's just me, but I'm seeing "Primary email updated" in the new plaintext? At least, in SB.

<img width="940" alt="image" src="https://user-images.githubusercontent.com/13018240/133519300-f423da2e-f26e-40ef-9733-eeb9565ba4da.png">

Reply:
good eye